### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by hillamu2**
pr

hillamu2/gci16.fossasia.org

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-- name: Animesh Singh
+
+=- name: Animesh Singh
```
Comment (jigyasa-grover) [inline _data/blogs.yml]: What's `=` doing here ?

**Comment by hillamu2** [comment]
what is a live link?

**Comment by oanarosca** [comment]
@hillamu2 it's the link to your fork's gh-pages. If you're not working on gh-pages, you can upload a screenshot so we can see your changes.

File: _data/blogs.yml [hunk 2]
```diff
@@ -55,3 +56,9 @@
   blog: https://wordpress.com/posts/jerry2001.school.blog
   article: GCI experience
   article_link: https://wordpress.com/post/jerry2001.school.blog/131
+
+- name: Hillary Moutou
+  blog : http://hillamu2.edublogs.org/
+  blog img :blogpost.png
```
Comment (jigyasa-grover) [inline _data/blogs.yml]: It rather should be `blog_img: blogpost.png`
Fix the space inconsistency.

Comment (hillamu2) [reply _data/blogs.yml]: thank you...can you please clarify what is the live link?

**Comment by hillamu2** [comment]
@oanarosca where do I get the live link ??

**Comment by m1guelpf** [comment]
@hillamu2 usually it is yourname.github.io/[repo-name]

**Comment by hillamu2** [comment]
@m1guelpf thank you!

**Comment by m1guelpf** [comment]
@hillamu2 Please fix the merge errors

**Comment by hillamu2** [comment]
@m1guelpf what's the problem?

**Comment by oanarosca** [comment]
@hillamu2 A merge conflict happens when things change as a result of us merging new PRs. See the contributing.md file for help on how to solve them, the commit again.

**Pull request closed without merging**